chess_env.py

```diff
@@ -18,20 +18,32 @@ def reset(self, state=None):
         """
         pass
 
-    def step(self, move):
+    def step(self, move): #move assumed to be in the following format: move = chess.Move.from_uci("g1f3")
         """
         Make a move in the environment.
         :param move:
         :return:
         """
-        pass
 
-    def get_possible_moves(self):
+        if move not in self.legal_moves: #check if the move is legal
+            print("This move is not a legal move")
+        else:
+            self.push(move)
+            print(self)
+
+        return None
+
+    def get_possible_moves(self): #returns a list
         """
         Get all legal moves from the current state.
         :return:
         """
-        pass
+        possible_moves = []
+
+        for move in self.legal_moves:
```

Comment:
this will work perfectly. A more "Pythonic" way to do this would be to say
return list(self.legal_moves). You can leave it as is if you would like though because it will work fine.

Reply:
also, make sure to create a separate branch off of develop when starting a new task. This should just be in your other pull request.